tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -116,12 +118,15 @@ def __init__(self,
         reserved_memory = max(1536, int(0.07 * node_memory) + 1)
         effective_node_memory = node_memory - reserved_memory
 
+        # Be sure to round up to nearest integer otherwise we might create
+        # fewer than self.partition_limit partitions.
+        partition_replicas_per_node = int(
+            numpy.ceil(
```

Comment:
prefer `math.ceil()`

Reply:
Though I don't quite understand the logic here: if the number supported is say 10.5 replicas per node, why it valid to round up and test if we support 11? I mean it probably works due to the "slack" of 32 above but ISTM round down was the right way?

Reply:
I picked `numpy` because it's already used in this file whereas `math` isn't. But you're right that this should use floor division :)

Reply:
> I picked numpy because it's already used in this file

Well that strikes me as a pre-exisitng bug then, but OK we don't need to change it :).